.github/workflows/auto-update-spec-repo-links.yml

```diff
@@ -0,0 +1,80 @@
+name: Auto-update spec repo links
+on:
+  schedule:
+    # hourly at minute 46
+    - cron: "46 * * * *"
+  workflow_dispatch:
+
+jobs:
+  check-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      current-version: ${{ steps.check-versions.outputs.current-version }}
+      latest-version: ${{ steps.check-versions.outputs.latest-version }}
+      already-opened: ${{ steps.check-versions.outputs.already-opened }}
+    steps:
+      - uses: actions/checkout@v4
+
+      - id: check-versions
+        name: Check versions
+        env:
+          GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+        run: |
+          current_version=$(grep "PREVIOUS_SPECIFICATION_VERSION=v.*" \
```

Comment:
Should this be getting the value of`LATEST_SPECIFICATION_VERSION` and replacing it on `PREVIOUS_SPECIFICATION_VERSION` as the guide said?

Reply:
I'm not sure I totally follow, I think I removed the need for that step by only maintaining a single version in the file (which aligns with the version currently used in the links)

Reply:
Ah I see it now. I somehow thought while initially reviewing that you only moved the `update_specification_version.sh` and didn't change it. All good, sorry for the confusion!